nessie/system_test.go

```diff
@@ -200,27 +95,36 @@ func setupTest(t *testing.T) (context.Context, logging.Logger, string) {
 	logger := logger.WithField("testName", t.Name())
 	repo := createRepo(ctx, t)
 	logger.WithField("repo", repo).Info("Created repository")
-
 	return ctx, logger, repo
 }
 
 func createRepo(ctx context.Context, t *testing.T) string {
 	name := strings.ToLower(t.Name())
-	repoPath := config.rawBucketPath + "/" + name
-
-	err := client.CreateRepository(ctx, &models.RepositoryCreation{
-		DefaultBranch:    masterBranch,
-		ID:               swag.String(name),
-		StorageNamespace: swag.String(repoPath),
-	})
+	storageNamespace := viper.GetString("storage_namespace")
+	repoStorage := storageNamespace + "/" + name
+
+	_, err := client.Repositories.CreateRepository(repositories.NewCreateRepositoryParamsWithContext(ctx).
+		WithRepository(&models.RepositoryCreation{
+			DefaultBranch:    masterBranch,
+			ID:               swag.String(name),
+			StorageNamespace: swag.String(repoStorage),
+		}), nil)
 	require.NoError(t, err, "failed to create repo")
-
 	return name
```

Comment:
You should return the repo name, no?

Reply:
adding specific log here and have logged on startup the storage namespace used in case nothing specific passed